build.gradle

```diff
@@ -120,14 +124,14 @@ allprojects {
     apply plugin: 'java'
 
     ext {
-        hibernateVersion = '5.2.16.Final'
-        elasticSearchVersion = '6.4.2'
+        hibernateVersion = '5.2.18.Final'
+        elasticSearchVersion = '6.8.4'
     }
 
     dependencies {
         testCompile "org.springframework.boot:spring-boot-starter-test:${springBootVersion}"
 
-        compile "com.fasterxml.jackson.core:jackson-databind:2.9.10"
+        compile "com.fasterxml.jackson.core:jackson-databind:2.9.10.1"
```

Comment:
may be bump to latest 2.10.0?

Reply:
Since this is a transitive dependency I wanted to keep the difference in version numbers as small as possible to minimise the possibility of an incompatibility.

We don't directly depend on jackson databind so I have replaced this direct dependency with a line in the spring dependency configuration section to ensure the correct version is used, if you're happy with this approach?

Reply:
Good spot @smathangi I got that wrong. I have reinstated the direct dependency, however version 2.10.0 has breaking changes and does not compile. I'd like to keep this PR scoped to the minimum change necessary to resolve our CVEs.